bson-adt-core/src/main/scala/adt/bson/BsonValue.scala

```diff
@@ -1,8 +1,8 @@
 package adt.bson
 
-import org.bson.BSON
+import org.bson.BsonType
 import org.bson.types.ObjectId
-import org.joda.time.DateTime
+import org.joda.time.{DateTime, DateTimeZone}
```

Comment:
consider using nscala-time which is a nice scala wrapper around joda time
https://github.com/nscala-time/nscala-time